src/open_inwoner/search/tests/test_page.py

```diff
@@ -129,3 +129,193 @@ def test_pagination_links(self):
             self.assertEqual(
                 link["href"], f"?query=content&tags={self.tag.slug}&page=2"
             )
+
+
+@tag("e2e")
+@tag("elastic")
```

Comment:
@pi-sigma This test is both e2e and elastic.. this might be a problem in our CI setup.

Reply:
I think I fixed it by adding `--exclude-tag=e2e` to the `elastic` test run, and adding the elasticsearch service to the `e2e` phase (which works because they also aren't run in parallel).

Reply:
In hindsight, the configuration with two exclusion tags was bound to create this issue, but I was too focused on making CI work for the tests we had at that time. Good to see you got it working though...